### PR TITLE
Feature/in537 yourorgpt2

### PR DESCRIPTION
**Pull request by GlenHani**

### CODE REVIEW

File: INZFS.MVC/Controllers/FundApplicationController.cs [hunk 1]
```diff
@@ -863,6 +863,21 @@ private ViewResult GetViewModel(Page currentPage, Field field)
...
+                    var addressInputModel = (AddressInputModel)model;
+                    if (!string.IsNullOrEmpty(model.DataInput))
+                    {
+                        var userAddress = model.DataInput.Split(',').ToList();
+                        addressInputModel.addresslist = userAddress;
+                        addressInputModel.addresslist[0] = addressInputModel.AddressLine1;
+                        addressInputModel.addresslist[1] = addressInputModel.AddressLine2;
+                        addressInputModel.addresslist[2] = addressInputModel.City;
+                        addressInputModel.addresslist[3] = addressInputModel.County;
+                        addressInputModel.addresslist[4] = addressInputModel.City;
```
Comment (BEISKRAM) [inline INZFS.MVC/Controllers/FundApplicationController.cs]: City seems to be mentioned twice, is this expected

Comment (GlenHani) [reply INZFS.MVC/Controllers/FundApplicationController.cs]: var userAddress = model.DataInput.Split(',').ToList();
                        addressInputModel.AddressLine1 = userAddress[0];
                        addressInputModel.AddressLine2 = userAddress[1];
                        addressInputModel.City = userAddress[2];
                        addressInputModel.County = userAddress[3];
                        addressInputModel.PostCode = userAddress[4];

Comment (GlenHani) [reply INZFS.MVC/Controllers/FundApplicationController.cs]: This is an old commit

**Review by lorenzoAtBEIS — APPROVED** [review]
Tested locally, all looks good. Made minor changes to address validations to reflect core address fields

**Review by RSA99 — APPROVED** [review]
Approved

**Review by BEISFathir — APPROVED** [review]
Approved

**Pull request merged**